parsers/CENACE.py

```diff
@@ -308,28 +317,32 @@ def fetch_consumption(
 
 
 if __name__ == "__main__":
```

Comment:
not sure what this long list of prints is doing, but I'm guessing it's irrelevant whether we add the updated exchange fetches e.g. MX->US-CAL-CISO

Reply:
These are just for de-bugging when running the parser file directly, you'll find them in a lot of the parsers but they are not used by the backend at all.

I just got annoyed with all the type issues 😅